browser/lib/util/webstorage.js

```diff
@@ -9,25 +10,26 @@ var WebStorage = (function() {
 	 * somewhat roundabout way. (If unsupported or no window object,
 	 * will throw on accessing a property of undefined) */
 	try {
-		window.sessionStorage.setItem(test, test);
-		window.sessionStorage.removeItem(test);
+		globalObject.sessionStorage.setItem(test, test);
+        globalObject.sessionStorage.removeItem(test);
 		sessionSupported = true;
 	} catch(e) {
 		sessionSupported = false;
 	}
 
 	try {
-		window.localStorage.setItem(test, test);
-		window.localStorage.removeItem(test);
+        globalObject.localStorage.setItem(test, test);
+        globalObject.localStorage.removeItem(test);
 		localSupported = true;
 	} catch(e) {
 		localSupported = false;
 	}
 
-	function WebStorage() {}
+
+    function WebStorage() {}
 
 	function storageInterface(session) {
-		return session ? window.sessionStorage : window.localStorage;
+		return session ? globalObject.sessionStorage : globalObject.localStorage;
```

Comment:
It is trivial to enable localstorage as I have a plugin that does it -- and quicker too; but for long term maintenance Simon is correct; it would be better to eliminate the any additional plugins/modules that you can.      

@mattheworiordan I will defer to whatever you prefer:
1. Use my localstorage plugin, no other changes needed.
2. Create a new WebStorage wrapper for NativeScript that uses the internal appsettings.